userguide/content/en/docs/adding-content/lookandfeel.md

```diff
@@ -22,26 +22,29 @@ Prism for highlighting.
 
 ## Project style files
 
-To customize your project's look and feel, create your own version of either or
-both of the following Docsy placeholder files (note the **`_project.scss`**
-suffixes):
-
-- [`assets/scss/`**`_variables_project.scss`**][_variables_project] is where you
-  add project-specific definitions of theme variables such as [site
-  colors](#site-colors), as well as any additional Bootstrap variable values you
-  want to set. You can find a list of Docsy's theme variables and their default
-  values in [`assets/scss/`**`_variables.scss`**][_variables].  For information
-  about other Bootstrap 5 variables, see [Variable defaults] and Bootstrap's
-  [`_variables.scss`] file.
-- [`assets/scss/`**`_styles_project.scss`**][_styles_project] is where you can
-  add your own custom SCSS styles, including overriding any of the styles in
-  Docsy's theme SCSS files.
-
-[_styles_project]: https://github.com/google/docsy/blob/main/assets/scss/_styles_project.scss
-[_variables_project]: https://github.com/google/docsy/blob/main/assets/scss/_variables_project.scss
-[_variables]: https://github.com/google/docsy/blob/main/assets/scss/_variables.scss
-[`_variables.scss`]: https://github.com/twbs/bootstrap/blob/v5.2.3/scss/_variables.scss
-[variable defaults]: https://getbootstrap.com/docs/5.3/customize/sass/#variable-defaults
+To customize your project's look and feel, create your own version of the
+following Docsy placeholder files (note the **`_project*.scss`** suffixes) and
+place them inside your project's `assets/scss/` folder:
+
+- **`_variables_project.scss`**[^vp] and<br>
```

Comment:
Is there a reason we've changed from directly linking to the files in the repo to footnotes? I think I find the links clearer, but maybe it's just a matter of preference.

Reply:
Actually, now that I've factored out the folder prefix, I can put back the links. Previously, the combination of link style, code font and bold for part of the path wasn't working.